feedhenry/fh-mbaas-client
# 🚀 fix npm install errors on node 10

To verify:

1. Run `npm install` in the master branch with node 10, you will see some errors in the output.
2. Checkout this branch and run it again, you should not see any errors.
